Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submissions-list/RefreshableSubmissionsList.tsx

```diff
@@ -1,51 +1,24 @@
-import React, { useCallback } from 'react';
-import isNil from 'lodash/isNil';
-
-import { useSubmissionsDetails } from '../../../hooks/submissions/use-submissions-details';
-import Button, { ButtonType } from '../../guidelines/buttons/Button';
+import React from 'react';
 
 import SubmissionsList, { ISubmissionsListProps } from './SubmissionsList';
 
-type IRefreshableSubmissionsListProps = ISubmissionsListProps
+interface IRefreshableSubmissionsListProps extends ISubmissionsListProps{
+    externalElements?: () => React.ReactElement;
+}
 
 const RefreshableSubmissionsList = ({
     items,
     selectedSubmission,
     className = '',
-}: IRefreshableSubmissionsListProps) => {
-    const {
-        state: { currentSubmission },
-        actions: { getSubmissionResults },
-    } = useSubmissionsDetails();
-
-    const submissionsReloadBtnClassName = 'submissionReloadBtn';
-
-    const handleReloadClick = useCallback(async () => {
-        if (isNil(currentSubmission)) {
-            return;
-        }
-
-        const { problem: { id: problemId }, isOfficial } = currentSubmission;
-
-        await getSubmissionResults(problemId, isOfficial);
-    }, [ currentSubmission, getSubmissionResults ]);
-
-    return (
-        <>
-            <SubmissionsList
-              items={items}
-              selectedSubmission={selectedSubmission}
-              className={className}
-            />
-
-            <Button
-              onClick={handleReloadClick}
-              text="Reload"
-              type={ButtonType.secondary}
-              className={submissionsReloadBtnClassName}
-            />
-        </>
-    );
-};
-
+    externalElements,
```

Comment:
Add default function to return base button
functionName should be a verb
example name - renderExternalElements
```

            <Button
              onClick={handleReloadClick}
              text="Reload"
              type={ButtonType.secondary}
              className={submissionsReloadBtnClassName}
            />
```